ci/dcos.sc

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env amm
+
+import ammonite.ops._
+import ammonite.ops.ImplicitWd._
+
+import scala.io.Source
+import scala.util.Try
+
+import $file.utils
+import utils.SemVer
+
+// output directory for all packages going to S3
+val PACKAGE_DIR: Path = pwd / 'target / 'universal
+// dcos catalog template directory
+val TEMPLATE_DIR: Path = pwd / 'dcos
+// versioned dcos catalog directory
+val UNIVERSE_DIR: Path = pwd / 'target / 'dcos
+
+@main
+def localBuild(version: String) {
```

Comment:
Let's use SemVer. We have a scopt parser for it.

Reply:
currently in `ci/pipeline`

```
implicit val SemVerRead: scopt.Read[SemVer] =
  scopt.Read.reads(SemVer(_))
```

Let's move to `utils` so it can be shared?

Reply:
perfect.. thanks @timcharper

Reply:
This is completely removed